pyccel/ast/cuda.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
```

Comment:
This file is not executable so this shebang is useless

Reply:
[d04d3b8](https://github.com/pyccel/pyccel-cuda/pull/42/commits/d04d3b8282bf5fbec3a314526218a60573395bf5)